pychromecast/controllers/multizone.py

```diff
@@ -2,27 +2,173 @@
 Controller to monitor audio group members.
 """
 import logging
+from uuid import UUID  # noqa: F401,E501 pylint: disable=unused-import
```

Comment:
Why is flake and pylint complaining about this import?

Reply:
Because you're not using it ?